bower_components/ckeditor/skins/husky/dialog.css

```diff
@@ -243,6 +243,15 @@ div.cke_disabled .cke_dialog_ui_labeled_content div * {
     width: 100%
 }
 
+.cke_dialog_ui_hbox_first {
+    display: none;
+}
+
+.cke_dialog_ui_hbox_last {
+    padding: 5px;
+    text-align: center;
+}
+

```

Comment:
@3rfan These are the changes I am talking about. They are not in the IE file, so why do they need to be here? If you check e.g. the overlay for inserting a link now in a browser like chrome you will recognize that there are just 2 form fields instead of 4.